pkg/importer/dataStream_test.go

```diff
@@ -115,18 +107,21 @@ func getFileSize(testfile string) (int, error) {
 	return int(fi.Size()), nil
 }
 
-func getTestFilePath(testfile string) string {
-	// CWD is set within go test to the directory of the test
-	// being executed, so using relative path
-	imageDir, _ := filepath.Abs(TestImagesDir)
-	return filepath.Join(imageDir, testfile)
+func readFile(fileName string) ([]byte, error) {
+	f, err := os.Open(fileName)
+	if err != nil {
+		return nil, err
+	}
+	defer f.Close()
+	result, err := ioutil.ReadAll(f)
+	return result, err
 }
 
-func getURLPath(testfile string) string {
+func getTestFilePath(testfile string) string {
 	// CWD is set within go test to the directory of the test
 	// being executed, so using relative path
 	imageDir, _ := filepath.Abs(TestImagesDir)
-	return "file://" + imageDir + "/" + testfile
+	return filepath.Join(imageDir, testfile)
```

Comment:
yo don't want to use Join with URLs because Join calls Clean which can remove double "/"es

Reply:
But these are no longer URLs, they are file paths, and thus I should call filepath.Join

Reply:
Right, sorry, I was reading too quickly...